vmdb/app/models/ems_kubernetes.rb

```diff
@@ -20,7 +27,8 @@ def self.raw_connect(hostname, port, api_version)
   def self.raw_api_endpoint(hostname, port)
     require 'uri'
 
-    uri = URI::HTTP.build(:path => "/api", :port => port.to_i)
+    port &&= port.to_i
+    uri = URI::HTTP.build(:path => "/api", :port => port)
```

Comment:
> port.try(:to_i)

Interesting.  I thought that since `nil.to_i == 0`, then `nil.try(:to_i)` would also be 0, but I just tested and it's nil.  Pretty cool...I'll change it.